File: despatch.WEB/app/src/swagger/models/user-room-response.ts
/* tslint:disable */
import { BaseEntityResponse } from './base-entity-response';
import { ContactResponse } from './contact-response';
import { RoomResponse } from './room-response';
import { UserResponse } from './user-response';
export interface UserRoomResponse extends BaseEntityResponse{
  contact?: ContactResponse;
  missedMessagesCount?: string;
  room?: RoomResponse;
  roomId?: string;
  user?: UserResponse;
  userId?: string;
}

File: despatch.Domain/Models/Default/Response/UserRoomResponse.cs
namespace despatch.Domain.Models.Default.Response
{
    public class UserRoomResponse : BaseEntityResponse
    {
        public string MissedMessagesCount { get; set; }

        public string UserId { get; set; }
        public string RoomId { get; set; }

        public UserResponse User { get; set; }
        public RoomResponse Room { get; set; }
        public ContactResponse Contact { get; set; }
    }
}
